preferencesfx/src/main/java/com/dlsc/preferencesfx/PreferencesFx.java

```diff
@@ -241,11 +238,18 @@ public PreferencesFx addEventHandler(EventType<PreferencesFxEvent> eventType, Ev
    *
    * @param eventType    the event type from which to unregister
    * @param eventHandler the handler to unregister
-   *
    * @throws NullPointerException if either event type or handler are {@code null}.
    */
   public PreferencesFx removeEventHandler(EventType<PreferencesFxEvent> eventType, EventHandler<? super PreferencesFxEvent> eventHandler) {
     preferencesFxModel.removeEventHandler(eventType, eventHandler);
     return this;
   }
+
+  public PreferencesFxView getPreferencesFxView() {
+    return new PreferencesFxView(preferencesFxModel, navigationView, breadCrumbView, categoryController);
+  }
+
+  public PreferencesFxModel getPreferencesFxModel() {
```

Comment:
Exposing this is a bit risky in my opinion, I would replace this with a method `saveSettings()` and `loadSettings()` maybe.

Reply:
👍